<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Upplay</class>
 <widget class="QMainWindow" name="Upplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>383</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Upplay Player</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../.designer/backup/icons/play.png</normaloff>../../.designer/backup/icons/play.png</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="horizontalSpacing">
           <number>4</number>
          </property>
          <property name="verticalSpacing">
           <number>0</number>
          </property>
          <item row="7" column="3" rowspan="2">
           <widget class="QPushButton" name="btn_stop">
            <property name="minimumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>28</width>
              <height>28</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>22</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLabel" name="lab_title">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="locale">
             <locale language="German" country="Germany"/>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhNone</set>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
            </property>
           </widget>
          </item>
          <item row="0" column="0" rowspan="7">
           <widget class="QPushButton" name="albumCover">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>95</width>
              <height>95</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>80</width>
              <height>80</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="1" colspan="2">
           <widget class="DirectSlider" name="songProgress">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>20</height>
             </size>
            </property>
            <property name="sizeIncrement">
             <size>
              <width>3</width>
              <height>0</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickInterval">
             <number>20</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0" rowspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetMaximumSize</enum>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="btn_bw">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/bwd.png</normaloff>:/icons/bwd.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_play">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>32</width>
                <height>34</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>32</width>
                <height>34</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="shortcut">
               <string/>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
              <property name="default">
               <bool>false</bool>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_fw">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>28</width>
                <height>28</height>
               </size>
              </property>
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>:/icons/fwd.png</normaloff>:/icons/fwd.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
              <property name="flat">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="3" rowspan="6">
           <widget class="DirectSlider" name="volumeSlider">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
            <property name="tracking">
             <bool>false</bool>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="invertedControls">
             <bool>false</bool>
            </property>
            <property name="tickInterval">
             <number>25</number>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="4">
           <widget class="Line" name="line">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>3</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>3</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="midLineWidth">
             <number>0</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="8" column="1" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="spacing">
             <number>2</number>
            </property>
            <item>
             <widget class="QLabel" name="curTime">
              <property name="font">
               <font>
                <pointsize>7</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::MinimumExpanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="maxTime">
              <property name="font">
               <font>
                <pointsize>7</pointsize>
               </font>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="btn_mute">
            <property name="maximumSize">
             <size>
              <width>28</width>
              <height>30</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/icons/vol_mute.png</normaloff>:/icons/vol_mute.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1" rowspan="2" colspan="2">
           <widget class="QLabel" name="lab_artist">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
            <property name="openExternalLinks">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="1" rowspan="2" colspan="2">
           <widget class="QLabel" name="lab_album">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>6</pointsize>
              <stylestrategy>PreferAntialias</stylestrategy>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1" rowspan="2" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="lab_version">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
                <stylestrategy>PreferAntialias</stylestrategy>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="lab_rating">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>8</pointsize>
                <weight>75</weight>
                <bold>true</bold>
                <stylestrategy>PreferAntialias</stylestrategy>
               </font>
              </property>
              <property name="autoFillBackground">
               <bool>false</bool>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="playlist_widget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="library_widget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>25</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFle">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionChange_Media_Renderer"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Playlist"/>
    <addaction name="actionLoad_Playlist"/>
    <addaction name="separator"/>
    <addaction name="action_Close"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_viewLibrary"/>
    <addaction name="action_viewSearchPanel"/>
    <addaction name="action_smallPlaylistItems"/>
    <addaction name="separator"/>
    <addaction name="action_Dark"/>
    <addaction name="action_Fullscreen"/>
    <addaction name="separator"/>
    <addaction name="action_Preferences"/>
</widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_help"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menuFle"/>
   <addaction name="menuView"/>
   <addaction name="menuPreferences"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="action_Close">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_Dark">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="action_help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="action_viewLibrary">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Library</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="action_Fullscreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fullscreen</string>
   </property>
  </action>
  <action name="action_Preferences">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="action_smallPlaylistItems">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Small Playlist Items</string>
   </property>
  </action>
  <action name="actionChange_Media_Renderer">
   <property name="text">
    <string>Change Media Renderer</string>
   </property>
  </action>
  <action name="action_viewSearchPanel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Search panel</string>
   </property>
  </action>
  <action name="actionSave_Playlist">
   <property name="text">
    <string>Save Playlist</string>
   </property>
  </action>
  <action name="actionLoad_Playlist">
   <property name="text">
    <string>Load Playlist</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DirectSlider</class>
   <extends>QSlider</extends>
   <header>GUI/player/DirectSlider.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
